-------------------------------------------------------------------------------
Test set: altunity.alttrashcatj.ShopTests
-------------------------------------------------------------------------------
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 2.391 s <<< FAILURE! - in altunity.alttrashcatj.ShopTests
altunity.alttrashcatj.ShopTests  Time elapsed: 2.391 s  <<< ERROR!
ro.altom.altunitytester.altUnityTesterExceptions.ConnectionException: Could not create connection to 192.168.11.209:13000
	at altunity.alttrashcatj.ShopTests.setUp(ShopTests.java:48)
Caused by: java.net.ConnectException: Connection refused: connect
	at altunity.alttrashcatj.ShopTests.setUp(ShopTests.java:48)

altunity.alttrashcatj.ShopTests  Time elapsed: 2.391 s  <<< ERROR!
java.lang.NullPointerException
	at altunity.alttrashcatj.ShopTests.tearDown(ShopTests.java:60)

